src/openrct2/ride/CableLift.cpp

```diff
@@ -272,10 +264,7 @@ static bool sub_6DF01A_loop(Vehicle* vehicle)
         vehicle->track_progress = trackProgress;
         moveInfo = vehicle_get_move_info(vehicle->TrackSubposition, vehicle->track_type, trackProgress);
         CoordsXYZ unk = { moveInfo->x, moveInfo->y, moveInfo->z };
```

Comment:
You can have the sum in the same line:
```suggestion
        auto unk = CoordsXYZ{ moveInfo->x, moveInfo->y, moveInfo->z } + vehicle->TrackLocation;
```